### PR TITLE
Fix for tl 0.15.1+dev

### PR DESCRIPTION
**Pull request by nagolove**
Hello! For revision 9de53e8d18145c21c8dabeedba253df74f262191 with teal `0.15.1+dev` writes:
```
tabular.tl|94 col 37| argument 2: got number, expected integer                                                                                                                                        
tabular.tl|239 col 46| cannot index object of type Output with number                                                                                                                                 
tabular.tl|269 col 8| using the '#' operator on this map will always return 0                                                                                                                         
tabular.tl|305 col 86| argument 2: got <unknown type>, expected {string}                                                                                                                              
```
I propose solution 68d2b842367a7c32af073880b907c67de1807362

### CODE REVIEW

File: tabular.lua [hunk 1]
```diff
@@ -236,7 +236,7 @@ show_as_columns = function(t, bgcolor, seen, ids, column_order, skip_header)
          local line = { draw.V }
          for _, cname in ipairs(column_names) do
             local row = columns[cname][i]
-            output_cell(line, cname, row and row[h] or "", bgcolor and colors[(i % #colors) + 1])
+            output_cell(line, cname, row and row[math.floor(h)] or "", bgcolor and colors[(i % #colors) + 1])
```
Comment (hishamhm) [inline tabular.lua]: redefining `row_heights` to be `{integer:integer}`  would remove the need for this `math.floor` call, right?

**Review by hishamhm — COMMENTED** [review]
Thank you for this PR! I left one small suggestion.

File: tabular.lua (hunk 1 above)
Comment (nagolove) [reply tabular.lua]: Fixed in 0e95d1d0876b83efbfe9eb2eb5f992f10281713a

Comment (hishamhm) [reply tabular.lua]: thank you!!

**Comment by hishamhm** [comment]
thank you for applying the change! could you push the change you made in 0e95d1d to the generated `tabular.lua` file as well? Then I think this PR will be ready to merge!

**Comment by nagolove** [comment]
Done.

**Comment by hishamhm** [comment]
@nagolove Thank you!

**Pull request merged**